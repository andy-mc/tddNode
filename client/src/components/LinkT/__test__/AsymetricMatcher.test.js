it('will check the matchers and pass', () => {
  const user = {
    age: 23,
    createdAt: new Date(),
    id: Math.floor(Math.random() * 20),
    name: 'LeBron James',
  };

  expect(user).toMatchSnapshot({
    age: 23,
    createdAt: expect.any(Date),
    id: expect.any(Number),
  });
});