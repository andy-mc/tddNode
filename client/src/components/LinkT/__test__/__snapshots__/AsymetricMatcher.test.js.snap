// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`will check the matchers and pass 1`] = `
Object {
  "age": 23,
  "createdAt": Any<Date>,
  "id": Any<Number>,
  "name": "LeBron James",
}
`;
